src/crystal/compiler_rt/i128_info.cr

```diff
@@ -1,3 +1,4 @@
+@[Extern]
```

Comment:
This should be attached to the `struct` itself and not the `module`.
ditto below